pkgs/applications/window-managers/sway/swayest-workstyle.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "swayest-workstyle";
+  version = "1.2.7";
```

Comment:
```suggestion
  version = "1.3.0";
```

Reply:
Wow that really just landed while I was packaging huh? Let me test and update.

Reply:
Addressed by latest rev which contains this: https://github.com/NixOS/nixpkgs/compare/e1c8162b15a41f493b80c1f92316f0129c507d05..8eb5b59515647e40cbddea5e1f88070511cf36a3
Let me know if you see anything else. Thanks!